Template/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,6 @@
+import XCTest
+@testable import {PROJECT}Tests
+
+XCTMain([
+          testCase({PROJECT}Tests.allTests),
```

Comment:
[SwiftLint](https://github.com/realm/SwiftLint) seems to be quite popular in the community. Haven't tried it myself yet though (it's still on my todo list).